src/Domain/Insights/CyclomaticComplexityIsHigh.php

```diff
@@ -21,20 +22,26 @@ public function hasIssue(): bool
 
     public function getTitle(): string
     {
-        return sprintf('Having `classes` with more than 5 cyclomatic complexity is prohibited - Consider refactoring');
+        return sprintf('Having `classes` with more than ' .
```

Comment:
When you use sprintf, you can use `%s` to avoid concatenation and add your value in parameters

```suggestion
        return sprintf(
            'Having `classes` with more than %s cyclomatic complexity is prohibited - Consider refactoring',
            $this->getMaxComplexity(),
        ); 
```

Reply:
Thanks for pointing out, I have changed it.